src/Package/MSBuild.VSSetup/files.swr

```diff
@@ -177,6 +177,7 @@ folder InstallDir:\MSBuild\Current\Bin\amd64
   file source=$(X86BinPath)Microsoft.Build.dll vs.file.ngenArchitecture=all
   file source=$(MSBuildConversionBinPath)Microsoft.Build.Engine.dll vs.file.ngenArchitecture=all
   file source=$(X86BinPath)Microsoft.Build.Framework.dll vs.file.ngenArchitecture=all
+  file source=$(X86BinPath)Microsoft.Build.Framework.tlb
```

Comment:
```suggestion
  file source=$(X64BinPath)Microsoft.Build.Framework.tlb
```

? Or is the `.tlb` not arch-specific?

Reply:
> Or is the .tlb not arch-specific?

It never even occurred to me that this was possible, but it makes sense, and potentially is a flaw in the current build scripts that the tlb produced will be whatever bitness the machine doing the build is. I'll update this to produce both types and see if there are differences.